MCP Demo (#132)
symfony/ai: demo/README.md

* The UI is coupled to a [Twig LiveComponent](https://symfony.com/bundles/ux-live-component/current/index.html), that integrates different `Chat` implementations on top of the user's session.
* You can reset the chat context by hitting the `Reset` button in the top right corner.
* You find three different usage scenarios in the upper navbar.

### MCP
Demo MCP server added with a `current-time` tool to return the current time, with an optional format string.

> To add the server, add the following to your `mcp.json` settings in your IDE:

Reviewer: ```suggestion
To add the server, add the following configuration to your MCP Client's settings, e.g. your IDE:
```

* The UI is coupled to a [Twig LiveComponent](https://symfony.com/bundles/ux-live-component/current/index.html), that integrates different `Chat` implementations on top of the user's session.
* You can reset the chat context by hitting the `Reset` button in the top right corner.
* You find three different usage scenarios in the upper navbar.

> ### MCP

Reviewer 2: ```suggestion
### MCP

```

Demo MCP server added with a `current-time` tool to return the current time, with an optional format string.

To add the server, add the following configuration to your MCP Client's settings, e.g. your IDE:
```json
{
    "servers": {
        "symfony": {
            "command": "php",
            "args": [
                "/your/full/path/to/bin/console",
                "mcp:server"
            ]
        }
    }
}
```

> #### Testing the MCP Server

Reviewer 2: ```suggestion
#### Testing the MCP Server

```
